OptimizelySDK.Net35/OptimizelySDK.Net35.csproj

```diff
@@ -173,7 +173,7 @@
     <Compile Include="..\OptimizelySDK\Utils\ExperimentUtils.cs">
       <Link>Bucketing\ExperimentUtils</Link>
     </Compile>
-    <Compile Include="..\OptimizelySDK\Bucketing\UserprofileUtil.cs">
+    <Compile Include="..\OptimizelySDK\Bucketing\UserProfileUtil.cs">
```

Comment:
Oh wow, nice fix. Does this mean that USPs were broken in Net35 and Net40 @msohailhussain ?

Reply:
yeah, but fixed here. 
https://github.com/optimizely/csharp-sdk/pull/128/commits/7503fbaf9536819e5dd5a4f10a475a76fa47f00b
When Audience PR will be merged, this will come as conflict.